test/extended/testdata/gssapi/fedora/kerberos_configured/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gssapiproxy/fedora-gssapi-kerberos
+
+ENV CLIENT CLIENT_HAS_LIBS_IS_CONFIGURED
+
+RUN sed -i.bak1 -e "s/EXAMPLE\.COM/${REALM}/g" /etc/krb5.conf && \
+    sed -i.bak2 -e "s/example\.com/${HOST}/g" /etc/krb5.conf && \
+    sed -i.bak3 -e "s/#//g" /etc/krb5.conf
```

Comment:
uncommenting all lines seems like it would be problematic... don't we want to be more judicious here?